tests/common/test_checked.py

```diff
@@ -307,6 +307,22 @@ def __hash__(self):
         )
         assert repr(tl) in {expected_template.format(e) for e in expected}
 
+    def test_common_checkedset_pickling(self):
+        StrSet = self.BaseSet[str]
+        sd = StrSet({"123", "456"})
+
+        assert sd.type is str
```

Comment:
If possible please update to use `self.assertIs` etc (just so that we don't forget)

Reply:
Will do.